test/helpers/engineConditions.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  EngineCondition,
+  EngineConditionQuery,
+  QueryFilterComparisonType,
+  PageView,
+  VectorQueryValue,
+  AudienceQueryDefinition,
+} from '../../types';
+
+export const makeQuery = <T extends AudienceQueryDefinition>(
+  queryValue: VectorQueryValue,
+  featureVersion: number,
+  queryFilterComparisonType: QueryFilterComparisonType
+) =>
+  ({
+    featureVersion,
+    queryProperty: 'topicDist',
+    queryFilterComparisonType,
+    queryValue,
+  } as EngineConditionQuery<T>);
+
+export const makeEngineCondition = <T extends AudienceQueryDefinition>(
```

Comment:
Is there not a way we could use the createCondition code here?

Feels like this and `makeQuery` duplicate what happens in the codebase?

Reply:
I'm sorry, I'm not sure about which `createCondition` method you're referring.
Is that the [evaluateCondition](https://github.com/AirGrid/edgekit/blob/develop/src/engine/evaluate.ts#L10) method (which was renamed from `createCondition`)?